eng/helix/helix.proj

```diff
@@ -21,11 +22,12 @@
     <IsExternal>true</IsExternal>
     <SkipInvalidConfigurations>true</SkipInvalidConfigurations>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
+    <HelixAccessToken Condition="'$(UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
```

Comment:
Suggest naming the environment variable `HelixAccessToken` to avoid this extra property.

Reply:
Nice, done

Reply:
This doesn't work, always setting access token has the effect of always using authenticated internal helix queues